storage/innobase/xtrabackup/test/t/ib_part_include.sh

```diff
@@ -34,15 +34,18 @@ ib_part_restore $TEST_VAR_ROOT/remote $mysql_datadir
 
 start_server
 # also test xtrabackup --stats work with --tables-file
-xtrabackup --stats --tables='test.test$' --datadir=$topdir/backup
 
-COUNT=`xtrabackup --stats --tables='test.test$' --datadir=$topdir/backup \
+ib_part_assert_checksum $checksum_a
+
+mysql -e "set global innodb_fast_shutdown=0"
+shutdown_server
+
+xtrabackup --stats --tables='test.test$' --datadir=$mysql_datadir
+COUNT=`xtrabackup --stats --tables='test.test$' --datadir=$mysql_datadir \
```

Comment:
you are changing from backup dir to server MySQL datadir

So PXB 8.0.30 is not able to read stats from backup dir. This can be added as restriction and the test can fail.

Or we can decide to fix later.

Please do not change to server datadir , keep the original testcaes as is (ie --datadir=topdir/backup)

Reply:
updated the test case.

Reply:
:+1: marking it as resolved